signer/src/lib.rs

```diff
@@ -99,6 +99,21 @@ pub const DEFAULT_MAX_DEPOSITS_PER_BITCOIN_TX: u16 = 25;
 /// smart contract.
 pub const DEPOSIT_DUST_LIMIT: u64 = 546;
 
+/// This is the least amount that can be withdrawn from Stacks onto
+/// bitcoin. The smart contract has a dust limit, but we have our own to
+/// make sure that we respect bitcoin's dust limits even if the smart
+/// contracts changed are updated and this check is removed.
+pub const WITHDRAWAL_DUST_LIMIT: u64 = 546;
+
+/// This is the number of bitcoin blocks that the signers will wait before
+/// acting on a withdrawal request. We do this to ensure that the
+/// withdrawal request is deemed final on the Stacks blockchain.
+pub const WITHDRAWAL_BLOCKS_WAIT: u64 = 6;
```

Comment:
Maybe `WITHDRAWAL_MIN_CONFIRMATIONS` would be more clear? (that's what I had used in #1321 anyway).

Reply:
I'll switch to that. I wasn't that happy with `WITHDRAWAL_BLOCKS_WAIT`.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1352/commits/c04dd3ee08723affcf5f186321fb01bda76493aa